src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea, DescriptionFigure {
+    protected String color;
```

Comment:
add getter and setter

Reply:
Can't get why we need getters and setters?
Why it works without Getters/Setters?

Reply:
because access modifier is protected

Reply:
if we have private access, without getters/setters we can't read/write this variable